static/css/mozsearch.css

```diff
@@ -75,9 +78,12 @@ table {
 @media (prefers-color-scheme: dark) {
   :root {
     --page-background: rgb(28, 27, 34);
-    --page-foreground: rgb(251, 251, 254);
+    --page-foreground: rgb(210, 211, 212);
```

Comment:
As per the bug comment, these come from our in-content pages so I'd rather keep foreground and background as-is unless there's a good reason not to do that.

Reply:
Hmm, ok, that's fair. FWIW GitHub also has a lighter foreground color value similar to Bugzilla. I personally find it more pleasant but I get the design system consistency aspect.

Reply:
reverted